Sources/String+Polymorphic.swift

```diff
@@ -85,6 +85,13 @@ extension String: Polymorphic {
     public var object: [String : Polymorphic]? {
         return nil
     }
+
+    /**
+        Converts the string to a UTF8 array of bytes.
+    */
+    public var bytes: [UInt8]? {
+        return [UInt8](self.utf8)
```

Comment:
utf8 is a little different in that it returns a sequence, but not an array. 

I like having polymorphic as an independent entity quite a bit, but this does bring up a good point about a potential conflict with Core if it defines the same variable.

I really wish there was a better way to disambiguate variables. 

I'd rather have it here since it adds more value to other polymorphic entities. What do you think about that.